src/plugins/CMakeLists.txt

```diff
@@ -1 +1,4 @@
 add_subdirectory( "bmqprometheus" )
+add_subdirectory( "bmqauthnbasic" )
+add_subdirectory( "bmqauthnpass" )
+add_subdirectory( "bmqauthnfail" )
```

Comment:
As the pool of plugins expands, we might want a doc (maybe just a readme in `src/plugins`?) that gives an overview of the various plugins that come out of the box and whether they're intended to be used in production vs serve as examples.